docs/configuration.md

```diff
@@ -333,3 +333,17 @@ ExecStart=/usr/bin/podman run \
 # Change your status port
 
 Use `SC4S_LISTEN_STATUS_PORT` to change the "status" port used by the internal health check process. The default value is `8080`.
+
+# Parallelize
+Similar to eBPF for TCP congestion of single heavy data stream we use parallelize feature.
+
```

Comment:
Use the parallelize feature to manage TCP congestion when using single heavy-data streams.